contributing/code/patches.rst

```diff
@@ -1,10 +1,21 @@
-Submitting a Patch
+Proposing a Change
 ==================
 
-Patches are the best way to provide a bug fix or to propose enhancements to
-Symfony.
+A pull request, "PR" for short, is the best way to provide a bug fix or to
+propose enhancements to Symfony.
 
-Step 1: Setup your Environment
+Step 1: Check existing Issues and Pull Requests
+-------------------------------------
```

Comment:
This line should be as long as its heading. Otherwise RST format will show an error. Thanks.

Reply:
thx. fixed